srcpkgs/mako/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'mako'
+pkgname=mako
+version=1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config scdoc wayland-protocols"
```

Comment:
add wayland-devel to hostmakedepends=

Reply:
Now I'm confused: should I add or exchange wayland-protocols and wayland-devel instead? Sorry, but communication sucks

Reply:
i didn't say anything about exchanging and removing both hostmakedepends and makedepends should have wayland-devel and wayland-protocols

Reply:
ok, now it should be fixed